ui/app/templates/vault/cluster/access/methods.hbs

```diff
@@ -69,36 +69,42 @@
       </div>
       <div class="level-right is-flex is-paddingless is-marginless">
         <div class="level-item">
-          <PopupMenu @name="auth-backend-nav">
-            <nav class="menu" aria-label="navigation for managing access method {{method.id}}">
-              <ul class="menu-list">
-                <li>
-                  <LinkTo @route="vault.cluster.access.method.section" @models={{array method.id "configuration"}}>
-                    View configuration
-                  </LinkTo>
-                </li>
-                {{#if method.canEdit}}
-                  <li>
-                    <LinkTo @route="vault.cluster.settings.auth.configure" @model={{method.id}}>
-                      Edit configuration
-                    </LinkTo>
-                  </li>
-                {{/if}}
-
-                {{#if (and (not-eq method.methodType "token") method.canDisable)}}
-                  <ConfirmAction
-                    @isInDropdown={{true}}
-                    @confirmTitle="Disable method?"
-                    @confirmMessage="This may affect access to Vault data."
-                    @buttonText="Disable"
-                    @onConfirmAction={{perform this.disableMethod method}}
-                  />
-                {{/if}}
-              </ul>
-            </nav>
-          </PopupMenu>
+          <Hds::Dropdown @isInline={{true}} @listPosition="bottom-right" as |dd|>
+            <dd.ToggleIcon
+              @icon="more-horizontal"
+              @text="Overflow options"
+              @hasChevron={{false}}
+              data-test-popup-menu-trigger
+            />
+            <dd.Interactive
+              @text="View configuration"
+              @route="vault.cluster.access.method.section"
+              @models={{array method.id "configuration"}}
+            />
+            {{#if (or method.canEdit (and (eq method.methodType "aws") method.canEditAws))}}
+              <dd.Interactive
+                @text="Edit configuration"
+                @route="vault.cluster.settings.auth.configure"
+                @model={{method.id}}
+              />
+            {{/if}}
+            {{#if (and (not-eq method.methodType "token") method.canDisable)}}
+              <dd.Interactive @text="Disable" @color="critical" {{on "click" (fn (mut this.methodToDisable) method)}} />
```

Comment:
to disable engines via the popup menu in tests `data-test-confirm-action-trigger` might need to be added here

Reply:
I don't think we disable via the popup menu in the tests (they are currently passing) but I'll check enterprise